data_validation/__main__.py

```diff
@@ -183,7 +183,6 @@ def build_config_from_args(args, config_manager):
     Args:
         config_manager (ConfigManager): Validation config manager instance.
     """
-    config_manager.append_calculated_fields(get_calculated_config(args, config_manager))
```

Comment:
I think moving this line down may affect regular row validation... since on line 296 we check 'if args.comparison_fields is not None' and get_calculated_config() populates comparison fields.

Reply:
Had to adjust the conditions a bit, the code for normal validation remains as it was before.
Fixed [here](https://github.com/GoogleCloudPlatform/professional-services-data-validator/pull/607/commits/f0eb38bd9e472989bcb8a8ed43a69350c1d28973#diff-ec3232fc84a73e24bf5cf68f7beb86e56e968c565509c7e07b420d026fd4f09f)